README.md

```diff
@@ -168,30 +266,29 @@ drwxr-xr-x  13 kellyegreene  staff   442B Jun  2 11:02 .git
 -rw-r--r--   1 kellyegreene  staff    11K Jun  2 11:21 README.md
 ```
 
-Notice that at the top of the file output there are a bunch of files that start with a `.`, like
-`.DS_Store`
-
-Files like `.DS_Store` are not listed. That's because files that start with a `.` are hidden files.
-Your `.bash_profile` is a hidden file in your home directory.  If you want to see the hidden files you can add the `a` flag to `ls` by typing `$ ls -a`.
+Notice that at the top of the file output there are a bunch of files that start with
+a `.`, like `.DS_Store`
```

Comment:
DS_Store is Mac specific, only there if the Finder window is open on that dir, etc. i'd rather talk about hidden files as a sep. readme where we focus on .bash_profile or etc.